### PR TITLE
mounts of the Han cavalry

### PR DESCRIPTION
**Pull request by indoptogopt**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml [hunk 1]
```diff
@@ -8,6 +8,12 @@
     <Requirements>
       <Techs datatype="tokens">phase_town</Techs>
     </Requirements>
+    <History>
+        Horses were vital for the Han military in combat against the nomadic horsemen of the North. Because combat-capable horses were, for this reason, a precious good, the possession and provision of horses for individuals were strictly regulated. There was, for example, a law in place that allowed families to exempt three men from military service if they could provide a strong, healthy horse instead.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml:12]: This first paragraph seems fragmented. It jumps from the Han needing horses to the Han regulating them to the Han having a rule to let them get warhorses, and there is no through line.

You could rewrite it, but personally I think most is cut and the sentence from the too short paragraph below is added, it makes a pretty strong paragraph.

```suggestion
       War horses were vital in the Han military's fight against the nomadic horsemen of the North. However, indigenous Chinese breeds were comparably small and too weak for use in battle. This forced the Han to resort to other means of obtaining horses for their cavalry.
```

File: 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml [hunk 2]
```diff
@@ -8,6 +8,12 @@
...
+        However, indigenous Chinese breeds were comparably small and too weak for battle. This forced the Han to resort to other means of obtaining horses for their cavalry.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml:13]:
```suggestion
```

File: 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml [hunk 3]
```diff
@@ -8,6 +8,12 @@
...
+        One possibility was through trading with various nomadic peoples. With Xiognu, for example, although their relations were generally hostile, the Han sometimes engaged in trade for livestock. The Wusun, on the other hand, another nomadic people, were close allies, and when forming the marriage alliance, the dowry even included 1,000 horses. If the Han couldn't trade for them, they took desired horses by force. The general Li Guangli famously conducted two very costly and dangerous exhibitions against the faraway Greco-Bactria to get hands on powerful horses — The War of the Heavenly Horses.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml:14]:
```suggestion
        One possibility was through trading with various nomadic peoples. The empire's relation with the Xiognu was generally hostile, but sometimes the Han could trade with them for livestock. The Wusun, on the other hand, were close allies with the Han, and when forming the marriage alliance, the dowry even included 1,000 horses. If the Han couldn't trade for them, they took desired horses by force. The general Li Guangli famously conducted two very costly and dangerous exhibitions against the faraway Greco-Bactria to get hands on powerful horses — The War of the Heavenly Horses.
```

Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml:14]: Could you go into a bit more detail with the marriage alliance? Not much is needed, just who married whom.

**Review by TheShadowOfHassen — COMMENTED** [review]
Content edit

File: 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml (hunk 3 above)
Comment (indoptogopt) [reply 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml]: Like this? Or do you want the names?

Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml]: That's good thanks.

File: 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml [hunk 4]
```diff
@@ -8,6 +8,12 @@
     <Requirements>
       <Techs datatype="tokens">phase_town</Techs>
     </Requirements>
+    <History>
+        War horses were vital in the Han military's fight against the nomadic horsemen of the North. However, indigenous Chinese breeds were comparably small and too weak for use in battle. This forced the Han to resort to other means of obtaining horses for their cavalry.
+        One possibility was through trading with various nomadic peoples. The empire's relation with the Xiognu was generally hostile, but sometimes the Han could trade with them for livestock. The Wusun, on the other hand, were close allies with the Han. And when forming the marriage alliance by sending a Han princess to marry the king of the Wusun, the dowry even included 1,000 horses. There were other nomaduc tribes supplying the Han with horses, but less relevant.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml:13]:
```suggestion
        One possibility was through trading with various nomadic peoples. The empire's relation with the Xiognu was generally hostile, but sometimes the Han could trade with them for livestock. The Wusun, on the other hand, were close allies with the Han. And when forming the marriage alliance by sending a Han princess to marry the king of the Wusun, the dowry even included 1,000 horses.
```

File: 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml [hunk 5]
```diff
@@ -8,6 +8,12 @@
...
+        One possibility was through trading with various nomadic peoples. The empire's relation with the Xiognu was generally hostile, but sometimes the Han could trade with them for livestock. The Wusun, on the other hand, were close allies with the Han. And when forming the marriage alliance by sending a Han princess to marry the king of the Wusun, the dowry even included 1,000 horses. There were other nomaduc tribes supplying the Han with horses, but less relevant. 
+        If the Han couldn't trade for them, they took desired horses by force. The general Li Guangli famously conducted two very costly and dangerous exhibitions against the faraway Greco-Bactria to get hands on powerful horses — The War of the Heavenly Horses.
+        And to ensure sufficient mounts in the long term, the Han government carried out large-scale horse-breeding programs. So large-scale, in fact, that the number of available horses rose to multiple hundreds of thousands. They built many designated horse breeding stations, especially in the North and also in the West, the most precarious regions of the empire. And the horses were under the care of professional horsekeepers.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml:15]:
```suggestion
        And to ensure sufficient mounts in the long term, the Han government carried out large-scale horse-breeding programs. So large-scale, in fact, that the number of available horses rose to multiple hundreds of thousands. They built many designated horse breeding stations, especially in the North and also in the West, the most precarious regions of the empire. And the horses were under the care of professional horse keepers.
```

**Review by TheShadowOfHassen — COMMENTED** [review]
Copedit.

**Pull request merged**